e2e/helpers.go

```diff
@@ -16,15 +16,21 @@
 package e2e
 
 import (
+	"strings"
 	"time"
 
 	"github.com/google/uuid"
+	"github.com/sirupsen/logrus"
 
 	apitypes "github.com/openclarity/vmclarity/api/types"
 	"github.com/openclarity/vmclarity/core/to"
 )
 
-const DefaultTimeout = 60 * time.Second
+const (
+	defaultTimeout  = 5 * time.Minute
+	defaultPeriod   = 5 * time.Second
+	scanStartOffset = 5 * time.Second
```

Comment:
```suggestion
	DefaultTimeout  = 5 * time.Minute
	DefaultPeriod   = 5 * time.Second
	
	fullScanStartOffset = 5 * time.Second
```

Reply:
Why do the first two constants need to be exported?

Reply:
they dont, its just that they are reused across e2e tests and it makes it clearer for usage

Reply:
ok, done!